### PR TITLE
Put more resources into terraform

### PR DESCRIPTION
**Pull request by davidstanke**

### CODE REVIEW

**Comment by davidstanke** [comment]
> Did you mean to include the terraform file with the output of your gcloud command? Keeping it around for reference?

Yep! exactly. I'll add a comment to that effect.

File: experimental/terraform/_misc/exported_config.tf [hunk 1]
```diff
@@ -0,0 +1,581 @@
+resource "google_bigquery_table" "" {
```
[the remark this answers is not in the document]
Comment (davidstanke) [reply experimental/terraform/_misc/exported_config.tf:1]: _interesting!_ As soon as (if?) I get the scheduled queries working, I'll try omitting these statements.

File: experimental/terraform/_misc/exported_config.tf [hunk 2]
```diff
@@ -0,0 +1,581 @@
...
+  project  = "fourkeys-000182"
+  schema   = "[{\"mode\":\"NULLABLE\",\"name\":\"source\",\"type\":\"STRING\"},{\"mode\":\"NULLABLE\",\"name\":\"deploy_id\",\"type\":\"STRING\"},{\"mode\":\"NULLABLE\",\"name\":\"time_created\",\"type\":\"TIMESTAMP\"},{\"mode\":\"REPEATED\",\"name\":\"changes\",\"type\":\"STRING\"}]"
+  table_id = "deployments"
+}
+resource "google_bigquery_dataset" "" {
+  access {
+    role          = "OWNER"
+    special_group = "projectOwners"
+  }
+  access {
```
[the remark this answers is not in the document]
Comment (davidstanke) [reply experimental/terraform/_misc/exported_config.tf:24]: ALL YOUR DORA ARE BELONG TO US.

[the remark this answers is not in the document]
Comment (davidstanke) [reply experimental/terraform/_misc/exported_config.tf:24]: Er, I mean, thanks for pointing that out. Removed.

**Review by dinagraves — APPROVED** [review]

**Pull request merged**